cli/cfncluster/cfnconfig.py

```diff
@@ -145,6 +145,19 @@ def __init__(self, args):
             sys.exit(1)
         self.parameters.append(('KeyName', self.key_name))
 
+        # Get the EC2 key path
+        try:
+            self.key_path = __config.get(self.__cluster_section, 'key_path')
```

Comment:
so we are making key_path mandatory field in config file. so when customer upgrade, they need to update there config file ? I don't recommend making it mandatory in config file. I have like 20 config file, as a customer i need to go and update in all those config (not a good customer experience even if i don't want to use login option)

Reply:
It's optional in the config file